Caseworker Clerk Home page top section

aliuk2012 opened the pull request:
Just removes the breadcrumbs and tabs. A "Claims" page heading added and tests updated to pass.

jsugarman commented on features/caseworker_claims_list.feature:
```diff
@@ -11,14 +11,14 @@ Feature: Caseworker claims list
-      And I click on the Completed Claims tab
-     Then I should see only my claims
-      And I should see the claims sorted by oldest first
+  #Scenario: View completed claims
```
can this scenario be re-written to be whatever you have now

jsugarman commented:
further to our discussion i think you you should comment out entire scenarios that are otherwise not exercising/testing any components and use a # TODO comment on each one for the time being so that, at a later stage, we can run "rake notes" to easily identify those scenarios.

aliuk2012 commented:
I've added the TODO notes and refactored the tests as discussed. BTW, running rake notes looks like it doesn't look at feature or spec folder. if you run rake notes now you will see that none of my TODO's are listed.
